test/system/600-completion.bats

```diff
@@ -231,6 +238,23 @@ function _check_completion_end() {
     is "${lines[-1]}" "Completion ended with directive: ShellCompDirective$1" "Completion has wrong ShellCompDirective set"
 }
 
+# check that there no suggestions in the output
+# we could only check stdout and not stderr but this is not possible with bats
+# by default we always have to extra lines at the end
+# and then we could have other extra lines for debugging, they will always start with [Debug]
+function _check_no_suggestions() {
+    if [ ${#lines[@]} -gt 2 ]; then
+        # checking for line count is not enough since we may include additional debug output
+        # lines starting with [Debug] are allowed
+        local i=0
+        length=$((${#lines[@]} - 2))
+        while [[ i -lt length ]]; do
+            assert "${lines[$i]:0:7}" == "[Debug]" "Suggestions are in the output"
```

Comment:
The error message might not be helpful to someone looking at the test failure. Suggestion:
```
.... "Unexpected non-Debug output line: ${lines[$i]}"
```
(Including `line` because otherwise the end user only sees the first seven characters)